cfnet/datasets.py

```diff
@@ -157,7 +157,9 @@ def __next__(self):
             if self.shuffle:
                 self.key = next(self.key_seq)
                 self.indices = jax.random.permutation(self.key, self.indices)
-            batch_data = [self.dataset[i] for i in self.indices[: self.batch_size]]
+            #batch_data = [self.dataset[i] for i in self.indices[: self.batch_size]]
+            batch_data = self.dataset[self.indices[: self.batch_size].tolist()]
```

Comment:
why `tolist()` is necessary here?

Reply:
I have deleted it, I used it to test the syntax.